_pages/about.md

```diff
@@ -4,13 +4,20 @@ title: About
 subtitle: Mila GFN Workshop
 permalink: /about
 ---
-<img src="{{ site.logo }}" style="width:300px;vertical-align:middle;padding-right:30px;float:left">
-Lorem ipsum
+<img src="{{ site.logo }}" style="width:400px;vertical-align:middle;padding-right:30px;float:left">
 
-Lorem ipsum
+Welcome to the GFlowNets workshop! During this workshop, we will explore the fascinating world of Generative Flow Networks (GFN). GFNs are a powerful framework that is related to concepts from Monte-Carlo Markov chain methods, reinforcement learning, generative models, and amortized variational methods. GFNs are designed to sample from a distribution specified by an energy function, allowing, allowing us to generate complex objects through a sequence of steps. They learn to represent and sample from a distribution by training a policy that generates objects with a probability proportional to a reward function or energy function.
+
+Throughout the workshop, we will discuss the backgrounds, theoretical foundations, techniques, and applications of GFNs, providing you with a comprehensive understanding of this exciting field. We hope you enjoy the workshop and find inspiration in the potential of Generative Flow Networks.
+
+---
+
+# Goal
+
+The goal of the workshop is to establish an inclusive atmosphere where researchers and practitioners from diverse backgrounds can come together to have a better understanding and deeper insights about the theory, methods, and applications of GFlowNets. The workshop will delve into several topics, such as pre-requisites, theoretical backgrounds, algorithm designs, and practical applications. Additionally, the workshop will scrutinize the limitations of present methods, benchmarks, and assumptions, and explore ways to enhance existing methods to facilitate wider applications in fields like molecular discovery, biological sequence design, and other related areas.
 
 ---
 
-### Territorial acknowledgement
+### Acknowledgement
 
-We, the organisers, acknowledge that the lands and waters where most of us live and where our universities are located---[Université de Montréal](https://www.umontreal.ca/en/indigenouspeoples/), [McGill University](https://www.mcgill.ca/circ/land-acknowledgement) and [Concordia University](https://www.concordia.ca/indigenous/resources/territorial-acknowledgement.html)---are the traditional and unceded territories of Indigenous peoples. The Kanien’kehá:ka (Mohawk) Nation call Tiohtiá:ke the site commonly known as Montréal. Since long before the arrival of European settlers, this territory has been and continues to be a gathering place for many Indigenous peoples. We respect the continued connections with the past, present and future in our ongoing relationships with Indigenous and other peoples within our communities.
+TBD.
```

Comment:
Did you remove this for a particular reason? I think it would be good to keep it, actually. It is common practice to have such territorial acknowledgements in events organised in Canada. See, for example: https://sites.google.com/mila.quebec/mila-intranet/mila-talent-community-programs/equity-diversity-and-inclusion/indigenous-realities